vnext/Shared/Networking/IHttpResource.h

```diff
@@ -95,6 +95,12 @@ struct IHttpResource {
   virtual void SetOnResponse(std::function<void(int64_t requestId, Response &&response)> &&handler) noexcept = 0;
   virtual void SetOnData(std::function<void(int64_t requestId, std::string &&responseData)> &&handler) noexcept = 0;
   virtual void SetOnData(std::function<void(int64_t requestId, folly::dynamic &&responseData)> &&handler) noexcept = 0;
+  virtual void SetOnIncrementalData(
```

Comment:
Add doc comment (like on SendRequest) describing the method and its parameters? It would go a long way to help maintainers (and code reviewers) understand this code.

Reply:
As this is a callback, perhaps add a typedef for it on which you can document the parameters without needing to redirect through the callback parameter?

Reply:
The typedef could then also be used to shorten the implementation in WinRTHttpResource.h

Reply:
Leaving typedefs out because of this PR's scope.